logging/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+find_package(Boost 1.54 REQUIRED COMPONENTS log)
+
+include_directories(${Boost_INCLUDE_DIRS})
+include_directories(${LEATHERMAN_CATCH_INCLUDE})
+include_directories(${LEATHERMAN_NOWIDE_INCLUDE})
+include_directories(${LEATHERMAN_LOCALE_INCLUDE})
+include_directories(${LEATHERMAN_LOGGING_INCLUDE})
+
+add_definitions("-DLEATHERMAN_LOGGING_NAMESPACE=\"leatherman.logging\"")
```

Comment:
This is gross, and needs to be done in each place that has a different logging context (my cfacter changes have it added in 3 places). I can definitely make a `leatherman_logging_context` macro, but that doesn't change the facter that each `.cc` file needs to know its own logging context somehow.